api/src/pcapi/alembic/versions/20231023T155523_a98c6b643561_create_index_on_last_validation_author_fkey_on_offer_table.py

```diff
@@ -0,0 +1,27 @@
+"""
+create index on last validation author fk on "offer" table
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = "a98c6b643561"
+down_revision = "6025fbc18ebb"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute(
+        """
+        CREATE INDEX IF NOT EXISTS "idx_offer_lastValidationAuthorUserId"
```

Comment:
Le `concurrently` est important, sinon ça prend un lock exclusif sur la table pendant toute la durée de la création de l'index. Cf. [la doc](https://www.postgresql.org/docs/16/sql-createindex.html#SQL-CREATEINDEX-CONCURRENTLY) pour plus de détails.

```suggestion
        CREATE INDEX CONCURRENTLY IF NOT EXISTS "idx_offer_lastValidationAuthorUserId"
```

Idem pour les index dans les autres tables.

Reply:
@dbaty C'est changé